### ISSUE TITLE
distribution of duck sizes does not follow the natural bell curve of specimen sizes

### ISSUE THREAD

**Issue by codahq** [6e38/flowduck#1]
https://www.youtube.com/watch?v=SO1NvBd72L4

**Comment by codahq** [comment]
https://www.youtube.com/watch?v=SwxdBiazu8M

**Comment by 6e38** [comment]
This is true. The distribution of ducks size is completely random and will likely be an even/flat distribution given sufficient samples.
